### PR TITLE
JSON env file to allow for wider support

### PR DESCRIPTION
**Pull request by AnthonyCrowcroft**

### CODE REVIEW

**Comment by AnthonyCrowcroft** [comment]
its all working as intended but having an issue writing a good test for the helper function

**Comment by AnthonyCrowcroft** [comment]
@andrewmclagan this is working a treat now the test coverage now includes mocks to check the file and I've removed the need to edit the index.html, this allows for people to keep JS in the application. I'm needing to proceed with using this fairly soon and was wondering when you'd be able to review this so that I can can make any changes you might request.

File: packages/node/dist/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-'use strict';
+import axios from 'axios';
```
Comment (andrewmclagan) [inline packages/node/dist/index.js]: Why are you importing axios?

**Comment by andrewmclagan** [comment]
Hi @AnthonyCrowcroft I have time to merge this now. Although there are some outstanding issues with this PR. Thanks for all your hard work.

- Why is  axios dependancy being used?
- Would you mind separating the types into another PR, just to be clean on sem-ver?

Thats all really. The idea of a json file is very sound indeed. We may in future have optional json and js files as there could be use cases where js is preferable.

File: packages/node/dist/index.js (hunk 1 above)
Comment (AnthonyCrowcroft) [reply packages/node/dist/index.js]: I'm importing axios so i can use the mock framework to add test coverage to the bind function, so by using the axios in the bind method i make a get request to a static env.json so i can then bind it correctly to the window without requiring to write functionality to the index.html . the types changes are just to reflect this additional method being introduced and the removal of the default export, so does require to be together. possibly if we moved this pull request onto like a next so i can use it to get my feature out and then in a week or so i can do a small refactor to the cli to allow people to choose between js or json.

**Comment by AnthonyCrowcroft** [comment]
@andrewmclagan replied to your code review. I'm leaning towards us trying a next build so you can test it out and so i can make it a togglable choice as to whether they use a js or json file. can you describe a usecase for js over json?

**Comment by AnthonyCrowcroft** [comment]
@andrewmclagan I'm also happy to remove the package.json and update the yarn.lock instead just realised that may be annoying.

**Comment by andrewmclagan** [comment]
Hello sorry this has been ignored. Would be happy if those are removed

**Pull request closed without merging**